widget/embedded/src/components/SwapsGroup/SwapsGroup.tsx

```diff
@@ -17,11 +17,11 @@ export function SwapsGroup(props: PropTypes) {
 
     const loadingGroups = [
       {
-        title: 'Today',
+        title: i18n.t('Today'),
         swaps,
       },
       {
-        title: 'Last month',
+        title: i18n.t('Last month'),
```

Comment:
i18n.t is here also in the jsx. line 38. do we need both of them?

Reply:
fixed